tests/foreman/ui/test_registration.py

```
@@ -0,0 +1,334 @@
+            },
+        )
+        cmd = session.host.get_register_command(
+            {
+                'general.host_group': hg_name,
+                'advanced.rex_interface': iface,
+                'general.insecure': True,
+            },
+            full_read=True,
+        )
+
+        assert hg_name in cmd['general']['host_group']
+        assert module_ak_with_cv.name in cmd['general']['activation_key_helper']
+        assert module_lce.name in cmd['advanced']['life_cycle_env_helper']
+        assert constants.FAKE_0_CUSTOM_PACKAGE in cmd['advanced']['install_packages_helper']
+
+
+@pytest.mark.tier3
+@pytest.mark.usefixtures('enable_capsule_for_registration')
+def test_global_re_registration_host_with_force_ignore_error_options(
+    session, module_activation_key, default_os, default_smart_proxy, rhel7_contenthost
+):
+    """If the ignore_error and force checkbox is checked then registered host can
+    get re-registered without any error.
+
+    :id: 8f0ecc13-5d18-4adb-acf5-3f3276dccbb7
+
+    :expectedresults: Verify the force and ignore checkbox options
+
+    :CaseLevel: Integration
+
+    :steps:
+        1. create and sync repository
+        2. create the content view and activation-key
+        3. open the global registration form and select --force and --Ignore Errors option
+        4. registered the host with generated curl command
+        5. re-register the same host again and check it is getting registered
+
+    :parametrized: yes
+    """
+    client = rhel7_contenthost
+    with session:
+        cmd = session.host.get_register_command(
+            {
+                'general.operating_system': default_os.title,
+                'general.capsule': default_smart_proxy.name,
+                'general.activation_keys': module_activation_key.name,
+                'general.insecure': True,
+                'advanced.force': True,
+                'advanced.ignore_error': True,
```

I think we should not enable the` ignore error` flag here, we don't  want to skip the errors occurring during registration

---

Generally you're right. But it's on purpose in this testcase... check test name and description.